src/main/java/io/reactivex/internal/operators/observable/ObservableDematerialize.java

```diff
@@ -18,9 +18,9 @@
 import io.reactivex.internal.disposables.DisposableHelper;
 import io.reactivex.plugins.RxJavaPlugins;
 
-public final class ObservableDematerialize<T> extends ObservableSource<Try<Optional<T>>, T> {
+public final class ObservableDematerialize<T> extends ObservableSource {
```

Comment:
The original `ObservableSource` has its use by mandating a constructor with an upstream source and may later support some graph discovery by providing a standard way of getting the upstream.